src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+
+type PeopleInfoProps = {
+  people: Person[];
+};
+
+export const PersonInfo: React.FC<PeopleInfoProps> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <tbody>
+      {people.map(person => (
+        <tr
+          data-cy="person"
+          key={person.slug}
+          className={classNames({
+            'has-background-warning': person.slug === slug,
+          })}
+        >
+          <td>
+            <PersonLink person={person} />
+          </td>
+
+          <td>{person.sex}</td>
```

Comment:
you can apply destructuring and get rid of abundant (and repetitive) syntax.

Reply:
++